src/loaders/utils/GetVersion.ts

```diff
@@ -0,0 +1,13 @@
+interface WindowLibVersion {
+	__fandom_jw_player_version: string;
+}
+
+declare let window: WindowLibVersion;
+
+export function getVideoPlayerVersion() {
+	return process.env.VIDEO_VERSION;
+}
+
+export function setVersionWindowVar() {
+	window.__fandom_jw_player_version = process.env.VIDEO_VERSION;
```

Comment:
Looks like this is only called in useEffects, so it shouldn't be called on the server-side and cause breaks. Just to be future proof, I would add a window undefined check. But for now, I think this should be ok.